contractsV5/KyberTradeLogic.sol

```diff
@@ -282,9 +285,12 @@ contract KyberTradeLogic is KyberHintHandler, IKyberTradeLogic, PermissionGroups
                 tData.tokenToEth.splitValuesBps,
                 tData.ethToToken.tradeType,
                 tData.ethToToken.addresses,
-                tData.ethToToken.splitValuesBps
+                tData.ethToToken.splitValuesBps,
+                failedIndex
             ) = parseHintT2T(hint);
         }
+
+        require(failedIndex > 0, "Hint error");
```

Comment:
notice for get rate should return 0 and not revert.
consult Desmond how to know which is the calling function.

Reply:
should probably return some error code from this function.
and calling function should return 0 rate.
trade knows to revert on 0 rate.
so this is how it should join the trade flow.